.eslintrc

```diff
@@ -17,7 +17,7 @@
         "printWidth": 80,
         "tabWidth": 2,
         "useTabs": false,
-        "trailingComma": "none",
+        "trailingComma": "es5",
```

Comment:
What's the difference between this and `"comma-dangle": "always-multiline"`?

https://eslint.org/docs/rules/comma-dangle#options

Reply:
They're the same. Always add trailing commas when using multiple lines, and not when on a single line. 

```js
// Doesn't get trailing comma
let contributors: ['adam', 'jonathan', 'steve', 'david']

// Trail those commas!
let contributors: [
  'adam',
  'jonathan',
  'steve',
  'david',
]
```